### ISSUE TITLE
From Dave B: way to sort cat #s using different fields?

### ISSUE THREAD

**Issue by Chovin** [OIT-UOG/specify-webportal-installer#94]
Occupation(s): **other**

---

Image thumbs are currently sorted by cat # by default, with no way to change. While not a huge deal, it would be helpful to be able to sort by species (actually, by genus + species), collection date, and maybe one or two other fields. The priority would be by species, though. Another option would be to display thumbnails according to how the records are sorted.

**Comment by Chovin** [comment]
so the image gallery shares the same results as the `table` tab, technically the functionality is there, you just need to switch tabs to sort it. I could look into putting this bar on top of the gallery tab as well though just for sorting:
![image](https://user-images.githubusercontent.com/17536161/96800393-17daeb80-13ba-11eb-98b8-007780e752ae.png)

**Comment by Chovin** [comment]
@burdickdr sorry, I forgot to ask for your feedback on this item

**Comment by burdickdr** [comment]
Yeah, I think being able to sort using that same bar at the top of the image gallery would be the best way to go. But if it gets too complicated I think the next best thing would be to just have the images displayed in the same order as the records, however people choose to sort those records before moving over to the gallery. Even better would be to have the images displayed in the same order as the records are sorted by default, but then have that bar at the top so people can sort the images however they want from within that tab. This is all bells and whistles at this point, though, so it's ok to put a pin in this one as well, especially if it will take more than an hour to do.
